platform.md

```diff
@@ -179,3 +198,37 @@ group = [
 Where:
 
 - Both `id` and `version` MUST be present for each buildpack object in a group.
+
+### source-metadata.toml (TOML)
```

Comment:
These two headings should be added to the TOC at the top of this file.

Reply:
Fixed in c1f1c74
* removed the json section, as requested by @sclevine above
* renamed this section to project-metadata.toml
* added project-metadata.toml to TOC